tests/unit/submitter/test_biostudies_submitter.py

```diff
@@ -1,85 +1,161 @@
+        with open("{0}{1}".format(dirname(__file__), file_path)) as file:
+            expected_payload = json.load(file)
+        return expected_payload
+
+    # def test_submit_project_converts_and_sends_project(self):
+    #     # Given
+    #     test_case = self.submission.map_ingest_entity(
+    #         make_ingest_entity(self.entity_type, random_id(), random_uuid())
+    #     )
+    #     # When
+    #     result, accession = self.submitter.send_entity(test_case, self.archive_type, self.error_key)
+    #     other_attributes = {}
+    #
+    #     # Then
+    #     self.converter.convert.assert_called_once_with(test_case.attributes, other_attributes)
+    #     self.submitter._submit_to_archive.assert_called_once_with(self.empty_project)
+    #     self.assertEqual(UPDATED_ENTITY, result)
+    #
+    # def test_submit_project_return_creation(self):
+    #     # Given
+    #     self.submitter._submit_to_archive = MagicMock(return_value={'accession': self.biostudies_accession})
+    #     test_case = self.submission.map_ingest_entity(make_ingest_entity(self.entity_type, random_id(), random_uuid()))
+    #     # When
+    #     result, accession = self.submitter.send_entity(test_case, self.archive_type, self.error_key)
+    #     # Then
+    #     self.assertEqual(CREATED_ENTITY, result)
+    #
+    # def test_submit_project_return_error(self):
+    #     # Given
+    #     error_msg = f'TestCaseError{random_id(3)}'
+    #     self.submitter._submit_to_archive = MagicMock(side_effect=Exception(error_msg))
+    #     test_case = self.submission.map_ingest_entity(
+    #         make_ingest_entity(self.entity_type, random_id(), random_uuid()))
+    #     # When
+    #     result, accession = self.submitter.send_entity(test_case, self.archive_type, self.error_key)
+    #     # Then
+    #     self.assertEqual(ERRORED_ENTITY, result)
+    #     self.assertDictEqual(test_case.get_errors(), {self.error_key: [f'BioStudies Error: {error_msg}']})
+    #
+    # def test_send_all_projects_sorts_projects_into_results(self):
+    #     # Given
+    #     self.submitter.send_entity = MagicMock(side_effect=[
+    #         [UPDATED_ENTITY, "1234"], [CREATED_ENTITY, "5678"], [ERRORED_ENTITY, "9012"]
+    #     ])
+    #     entities = self.map_random_entities(self.submission, self.entity_type, 3)
+    #     calls = []
+    #     call_count = 0
+    #     for entity in entities:
+    #         call_count += 1
+    #         calls.append(call(entity, self.archive_type, self.error_key, {}))
+    #
+    #     # When
+    #     result, accessions = self.submitter.send_all_projects(self.submission)
+    #
+    #     # Then
+    #     self.submitter.send_entity.assert_has_calls(calls, any_order=True)
+    #     self.assertEqual(call_count, self.submitter.send_entity.call_count)
+    #     self.assertEqual(1, len(result[UPDATED_ENTITY]))
+    #     self.assertEqual(1, len(result[CREATED_ENTITY]))
+    #     self.assertEqual(1, len(result[ERRORED_ENTITY]))
```

Comment:
why are these commented out?

Reply:
Accidentally forgot to delete it. It is done now. Tx